src/utils/fetchClient.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const BASE_URL = 'https://backend-phone-store.onrender.com/';
+
+async function request<T>(searchParams: string, method = 'GET'): Promise<T> {
+  const options: RequestInit = { method };
+
+  try {
+    const response = await fetch(BASE_URL + searchParams, {
+      headers: {
+        'Access-Control-Allow-Origin': 'http://localhost:3000',
```

Comment:
check this once more because we didn't use this header before and we will not use localhost at all